### DISCUSSION TITLE
Smallrye Reactive Messaging AMQP tracing

### DISCUSSION THREAD

**Discussion by serhiynovos** [quarkusio/quarkus#29544] [Q&A]
Hi. Does AMQP connector supports tracings ? I see Kafka connector does support it but I did not found any information about AMQP

[a comment by an automated account is not shown]

**Reply by radcortez** [reply]
Yes. AMQP also supports it. Check the configuration reference: https://smallrye.io/smallrye-reactive-messaging/smallrye-reactive-messaging/3.6/amqp/amqp.html#_configuration_reference.

You need to have `tracing-enabled` enabled (which should be, since it is the default). 

Are you having any issues getting traces from the AMQP Connector?

**Reply by serhiynovos** [reply]
I was trying before to add traces, for Kafka it worked fine, but not for amqp. AFAIK currently for quarkus are two tracing libraries and one of them is deprecated, right?

As I remember I enabled tracing in properties but without any luck

**Reply by radcortez** [reply]
You also need the OpenTelemetry Extension.
